doc/source/reference/algorithms.centrality.rst

```diff
@@ -57,17 +57,15 @@ Eigenvector
    katz_centrality
    katz_centrality_numpy
 
-Communicability
+Subgraph
 ---------------
 .. autosummary::
    :toctree: generated/
 
-   communicability
-   communicability_exp
-   communicability_centrality
-   communicability_centrality_exp
-   communicability_betweenness_centrality
-   estrada_index
+    subgraph_centrality
```

Comment:
One too many spaces in the indentation. Should be aligned with the first "a" on line 62.